native/cocos/bindings/manual/jsb_global.cpp

```diff
@@ -583,8 +585,9 @@ bool jsb_global_load_image(const ccstd::string &path, const se::Value &callbackV
 
                 if (loadSucceed) {
                     se::HandleObject retObj(se::Object::createPlainObject());
-                    dataVal.setUint64(reinterpret_cast<uintptr_t>(imgInfo->data));
-                    retObj->setProperty("data", dataVal);
+                    auto *obj = se::Object::createObjectWithClass(__jsb_cc_JSBNativeDataHolder_class);
+                    obj->setPrivateData(ccnew cc::JSBNativeDataHolder(imgInfo->data));
```

Comment:
```c++
obj->setPrivateData(ccnew cc::JSBNativeDataHolder(imgInfo->data));
```
needs to changed to 
```c++
auto *obj = JSB_MAKE_PRIVATE_OBJECT(cc::JSBNativeDataHolder, imgInfo->data);
s.thisObject()->setPrivateObject(obj);
```

Reply:
what's the difference?

Reply:
cannot get `se::state s` paramater in this funciton